composer.json

```diff
@@ -17,6 +17,7 @@
 	},
 	"require": {
 		"php": ">=7.4 <=8.1",
+		"ext-openssl": "*",
```

Comment:
This was recommended by PHPStorm and can be reverted if necessary. According to the Nextcloud admin docs, openssl is required in order to run the server. This is more of a cosmetic thing.